sites/fast-website/src/app/data/community.data.ts

```diff
@@ -0,0 +1,38 @@
+export interface CommunityContentPlacementData {
+    iconSrc: string;
+    iconAlt: string;
+    header: string;
+    body: string;
+    actionText: string;
+    actionLink: string;
+}
+
+export const communityContentPlacementData: CommunityContentPlacementData[] = [
+    {
+        iconSrc: "https://via.placeholder.com/16",
+        iconAlt: "Discord",
+        header: "Discord",
+        body:
+            "Join our active community on Discord. Follow the latest updates and contributions, ask questions, give feedback, or keep up on our reading list.",
+        actionText: "Join the Discord Chat",
+        actionLink: "#",
```

Comment:
Could we include the actual URLs here?  Discord = "https://discord.gg/FcSNfg4", Twitter can be found on current site.

Reply:
yes! I will add those.